esx_service/vmodl/Makefile

```diff
@@ -70,11 +72,17 @@ test: deploy
         set +x ; exit $$status'
 
 copy_to_payload:
-ifdef INSTALL_VMODL
 	echo GENERATING VMODL PAYLOAD...
 	mkdir -p ../$(VMDKOPS_PAYLOAD)$(TARGET2) ../$(VMDKOPS_PAYLOAD)$(TARGET3)
 	cp $(PYFILES) $(TESTFILES) ../$(VMDKOPS_PAYLOAD)$(TARGET2)
 	cp $(PYFILES) $(TESTFILES) ../$(VMDKOPS_PAYLOAD)$(TARGET3)
+
 else
-	@echo Skipping VMODL payloads. See https://github.com/vmware/docker-volume-vsphere/pull/975 for details.
+
+deploy test copy_to_payload:
```

Comment:
to make sure misc. CI scripts invoking 'make test' or 'make deploy' or 'make copy_to_payload' complete successfully with the proper message